src/msg/async/Event.cc

```diff
@@ -154,6 +163,16 @@ int EventCenter::init(int nevent, unsigned center_id, const std::string &type)
   if (!driver->need_wakeup())
     return 0;
 
+#ifdef __linux__
+  int efd = eventfd(0, EFD_NONBLOCK|EFD_CLOEXEC);
+  if (efd < 0) {
+    int e = ceph_sock_errno();
```

Comment:
`ceph_sock_errno()` is an API to hide the difference between W32 socket API and POSIX API which exposes errors using `errno`, and it should be used with the socket API. in this case, i'd suggest just check `errno` instead, as `eventfd()` is a linux-only API.

despite that we are using `ceph_sock_errno()` elsewhere in this source file, i think the reason is that we wanted to simplify the error handling where we use W32 socket API for notifications on Win32 platforms.

Reply:
Changed to just `errno`.